staticjinja/staticjinja.py

```diff
@@ -398,11 +398,13 @@ def get_dependencies(self, filename):
 
         :param filename: the name of the file to find dependencies of
         """
-        if self.is_partial(filename):
+        jinja2_filename = filename.replace(os.path.sep, "/")
```

Comment:
Would you be OK simplifying `jinja2_filename` to just `template_name`? Then it matches the other functions and the jinja convention, and I think it's more appropriate anyways.